docs/ConvertBMWToJSON.md

```diff
@@ -0,0 +1,45 @@
+# Convert BMW Encoding to JSON Structure
+
+This document describes steps for converting digitalized BMW encoding documents to a JSON file
+that will serve as the foundation for implementing the BMW input method.
+
+BMW encoding documents are in PDF format. These PDFs are composed by digitalized images of orginal
+books. The coversio method is:
```

Comment:
Typo coversio -> conversion - 
Also "digitized" is better rendering of "digitalized"